QUEST_LV_0100.tsv

```diff
@@ -1415,100 +1415,100 @@ QUEST_LV_0100_20150317_001414	{memo X}The Barrier Device is on the left end of h
 QUEST_LV_0100_20150317_001417	{memo X}Let's go to the next floor soon. {nl} Goddess Gabija is awaiting us. {nl}
-QUEST_LV_0100_20150317_001418	Refugee
-QUEST_LV_0100_20150317_001419	I lost everything. {nl} I don't know how I can go on anymore. {nl} Would my life have become this miserable if not for Medzio Diena?
-QUEST_LV_0100_20150317_001420	Four years ago, it was on the eve of the so called 'Medzio Diena'. {nl} Suddenly, a huge tree emerged in the middle of the capital and the earth and sky reversed. {nl}
-QUEST_LV_0100_20150317_001421	I never knew a tree could grow that big. {nl} And the destruction of the capital began. {nl}
-QUEST_LV_0100_20150317_001422	Everything happened so fast, and nobody knew what to do. {nl} We didn't know whether to run away or try to stop it... Our minds just became blank.
-QUEST_LV_0100_20150317_001423	The destruction of the capital was just the beginning. {nl} Medzio Diena was simply a prelude to the catastrophes coming ahead. {nl}
-QUEST_LV_0100_20150317_001424	Monstrous plants sprouted up all over and began devouring people.{nl} Everyone started running, but before I knew it I was the only person left.{nl}
-QUEST_LV_0100_20150317_001425	Enough. Don't ask any more questions. {nl}I.. don't want to think about it anymore.
-QUEST_LV_0100_20150317_001426	From here, it's the Research Camp on Zachariel's Royal Mausoleum.{nl}You can freely enter, but please be careful not to disturb the people doing research.
-QUEST_LV_0100_20150317_001427	$Niels is the one who is in charge of this place.{nl}If you need to ask something, go find him at the Research Camp.
+QUEST_LV_0100_20150317_001418	$Refugee
+QUEST_LV_0100_20150317_001419	$I lost everything. {nl}I don't know how I can go on anymore. {nl}Would my life have become this miserable if not for Medzio Diena?
+QUEST_LV_0100_20150317_001420	$Four years ago, it was on the eve of the so called 'Medzio Diena'. {nl}Suddenly, a huge tree emerged in the middle of the capital and the earth and sky turned {nl}upside down. {nl}
+QUEST_LV_0100_20150317_001421	$I never knew a tree could grow that big. {nl}And the destruction of the capital began. {nl}
+QUEST_LV_0100_20150317_001422	$Everything happened so fast, and nobody knew what to do. {nl}We didn't know whether to run away or try to stop it... Our minds just became blank.
+QUEST_LV_0100_20150317_001423	$The destruction of the capital was just the beginning. {nl}Medzio Diena was simply a prelude to the catastrophes coming ahead. {nl}
+QUEST_LV_0100_20150317_001424	$Monstrous plants sprouted up all over and began devouring people.{nl}Everyone started running, but before I knew it I was the only person left.{nl}
+QUEST_LV_0100_20150317_001425	$Enough. Don't ask any more questions. {nl}I... don't want to think about it anymore.
+QUEST_LV_0100_20150317_001426	$The Excavation Camp on Zachariel's Royal Mausoleum is past here.{nl}You can freely enter, but please be careful not to disturb the people doing research.
```

Comment:
Did you change replace "Research Camp" with "Excavation Camp" in every single file?